packages/@tinacms/cli/src/cmds/init/apply.ts

```diff
@@ -538,11 +538,13 @@ const logNextSteps = ({
       )
       logger.info(indentText(envFileText))
     }
+    logger.info(
+      'Before you can start your site you will need to update your site to use the backend client.\nSee docs for more info: https://tina.io/docs/self-hosted/querying-data/'
```

Comment:
"your site" is repetitive

Reply:
Update to "'Before you can run your site you will need to update to use the backend client"